examples/demo.rs

```diff
@@ -0,0 +1,56 @@
+#![allow(unused)]
+use std::io::{self, Read};
+use std::io::prelude::*;
+use std::time::{SystemTime,UNIX_EPOCH};
+use std::net::TcpStream;
+use std::{thread, time::Duration};
+use std::io::BufReader;
```

Comment:
```suggestion
use std::io::{self, Read, BufReader, prelude::*};
use std::time::{SystemTime, UNIX_EPOCH, Duration};
use std::net::TcpStream;
use std::thread;
```

Nit: imports can become an eye sore, but this is just my opinion 😅

Reply:
This should be solved by use of an LSP tool like rust-analyzer